### DISCUSSION TITLE
Modify K8s CIS Benchmark Policies

### DISCUSSION THREAD

**Discussion by Zowzow92** [aquasecurity/trivy#5415] [Q&A] [answered]
### Question

Hi everyone,

I would like to modify CIS Kubernetes Benchmark Policies. I tried to change policies located in ".cache" of my computer but it doesn't change the policies used when scan again my kube cluster. 
I also searched inside trivy.db but nothing about compliance inside of it ... only CVEs.

Does anyone has an idea of how I could test my own policies using the compliance K8s mode without PR Trivy Policy repo ?

Thanks

### Target

Kubernetes

### Scanner

Misconfiguration

### Output Format

None

### Mode

Standalone

### Operating System

Ubuntu

### Version

```bash
Version: 0.46.0
```

**Comment by itaysk** [comment, marked as answer]
1. copy the compliance definition from here: https://github.com/aquasecurity/trivy-policies/tree/main/rules/specs/compliance
2. change it however you want. you might need to create [custom checks](https://aquasecurity.github.io/trivy/v0.46/docs/scanner/misconfiguration/custom/)
4. load it as [custom compliance](https://aquasecurity.github.io/trivy/v0.46/docs/compliance/compliance/#custom-compliance) with `--compliance` flag.

**Reply by Zowzow92** [reply]
Thanks for your help @itaysk ❤️
